tests/unit/jobs/test_views.py

```diff
@@ -835,9 +837,118 @@ def test_get_submission_by_pk_when_user_is_neither_challenge_host_nor_submission
         self.url = reverse_lazy('jobs:get_submission_by_pk',
                                 kwargs={'submission_id': self.submission.id})
 
-        expected = {'error': 'Sorry, you are not authorized to access this submission.'}
+        expected = {
+            'error': 'Sorry, you are not authorized to access this submission.'}
```

Comment:
@Ayukha Can you please revert the changes above this because they are not required for this PR.